sys/fs/devfs/auto_init_devfs.c

```diff
@@ -30,8 +30,21 @@ static vfs_mount_t _devfs_auto_init_mount = {
     .mount_point = "/dev",
 };
 
+#ifdef FEATURE_PERIPH_HWRNG
+#include "periph/hwrng.h"
+
+static devfs_t random_devfs = {
+    .path = "/urandom",
```

Comment:
I would prefer /hwrng for this device, and that urandom uses the default PRNG to generate random data. Seeding the PRNG from the hwrng is a separate matter though.